app/controllers/message_objects_controller.rb

```diff
@@ -26,6 +26,9 @@ def show
 
   def download
     authorize @message_object
+
+    EventBus.publish(:form_object_downloaded, @message_object.message) if @message_object.form?
```

Comment:
Preco len form?

Reply:
@luciajanikova prave toto mi pride take zvlastne, ze je to akcia "download" ale nepusti sa vzdy, ja by som to odpalil vzdy ked stlacis download.

Reply:
upravila som implementaciu